tutorials/publish/index.md

```diff
@@ -198,14 +206,37 @@ Running the service and `SNET Daemon`.
 
 In the service folder, create a file named `snetd.config.json`. 
 
+```json
+{
+   "DAEMON_END_POINT": "SERVICE_IP:7000",
+   "ETHEREUM_JSON_RPC_ENDPOINT": "https://kovan.infura.io",
+   "IPFS_END_POINT": "http://ipfs.singularitynet.io:80",
+   "REGISTRY_ADDRESS_KEY": "0xe331bf20044a5b24c1a744abc90c1fd711d2c08d",
+   "PASSTHROUGH_ENABLED": true,
+   "PASSTHROUGH_ENDPOINT": "http://localhost:7003",
+   "ORGANIZATION_ID": "ORGANIZATION_ID",
+   "SERVICE_ID": "example-service",
+   "PAYMENT_CHANNEL_STORAGE_SERVER": {
+       "DATA_DIR": "/opt/singnet/etcd/"
+   },
+   "LOG": {
+       "LEVEL": "debug",
+       "OUTPUT": {
+              "TYPE": "stdout"
+           }
+   }
+}
+```
+
```

Comment:
I would propose to not duplicate content of `snetd.config.json`. And since it is not so obvious how to create a file inside a docker container I would simply leave:
```
cat > snetd.config.json << EOF
{
   "DAEMON_END_POINT": "0.0.0.0:7000",
   "ETHEREUM_JSON_RPC_ENDPOINT": "https://kovan.infura.io",
   "IPFS_END_POINT": "http://ipfs.singularitynet.io:80",
   "REGISTRY_ADDRESS_KEY": "0xe331bf20044a5b24c1a744abc90c1fd711d2c08d",
   "PASSTHROUGH_ENABLED": true,
   "PASSTHROUGH_ENDPOINT": "http://localhost:7003",
   "ORGANIZATION_ID": "ORGANIZATION_ID",
   "SERVICE_ID": "example-service",
   "PAYMENT_CHANNEL_STORAGE_SERVER": {
       "DATA_DIR": "/opt/singnet/etcd/"
   },
   "LOG": {
       "LEVEL": "debug",
       "OUTPUT": {
              "TYPE": "stdout"
           }
   }
}
```

However I do not insist...

Reply:
I don't know if my changes kept it clear, what do you think?

Reply:
yes... I think it is more or less clear...